awx/main/conf.py

```diff
@@ -164,6 +165,18 @@ def _load_default_license_from_file():
     read_only=True,
 )
 
+register(
+    'DEFAULT_EXECUTION_ENVIRONMENT',
+    field_class=fields.PrimaryKeyRelatedField,
+    allow_null=True,
```

Comment:
The most recent `managed_by_tower = True` version would be used.

Reply:
So then we still need some code to pre-create a default EE? Or is that already in the branch somewhere?

Reply:
Yes, we still need it and it isn't in-place yet.

Reply:
What if, instead of creating it in a migration of create_preload_data (I do not believe the latter is reliable), we create the default execution environment ad-hoc if a job tries to launch while no `managed_by_tower` execution environments exist? Just thinking that might be easier and more reliable.